Code/Framework/AzCore/AzCore/Serialization/Json/AnySerializer.cpp

```diff
@@ -63,17 +63,22 @@ namespace AZ
                     "The C++ declaration may have been deleted or changed.");
             }
 
-            result.Combine(ContinueLoadingFromJsonObjectField
-                ( AZStd::any_cast<void>(outputAnyPtr)
-                , anyTypeId
-                , inputValue
-                , "value"
-                , context));
+            // Allowing the serializer to read from objects contained in members "Value" and "value" for backward compatibility
+            if (inputValue.HasMember("value"))
+            {
+                result.Combine(
+                    ContinueLoadingFromJsonObjectField(AZStd::any_cast<void>(outputAnyPtr), anyTypeId, inputValue, "value", context));
+            }
+            else if (inputValue.HasMember("Value"))
+            {
+                result.Combine(
+                    ContinueLoadingFromJsonObjectField(AZStd::any_cast<void>(outputAnyPtr), anyTypeId, inputValue, "Value", context));
+            }
```

Comment:
Before, if the JSON didn't have a "value" field, this would cause the processing result to be halted and stop serialization.

With this change now, if it doesn't have a "value" or "Value" field, then the result code doesn't get updated.
This means the serialization would report success.

Reply:
Updated after offline discussion. Thanks!